gef.py

```diff
@@ -4078,75 +4056,102 @@ def get_insn_size(self, addr):
 
 
     def do_invoke(self, argv):
-        retval = None
-        perm_mode = False
-        opts, args = getopt.getopt(argv, "r:ph")
-        for o,a in opts:
-            if   o == "-r":
-                retval = long(a, 16)
-            elif o == "-p":
-                perm_mode = True
+        opts, args = getopt.getopt(argv, "b:h")
+        num_bytes = 0
+        for o, a in opts:
+            if o == "-b":
+                num_bytes = long(a, 0)
```

Comment:
changing from 16 to 0 allows the user to provide hex OR decimal (but hex must be precede with `0x`)